parser_example_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package parser_test
+
+import (
+	"fmt"
+	"github.com/pingcap/parser"
+	// 0. import parser_driver implemented by TiDB(user also can implement own driver by self).
+	_ "github.com/pingcap/tidb/types/parser_driver"
+)
+
+// This example show how to parse a text sql into ast.
+func Example_parseSQL() {
+
+	// 0. make sure import parser_driver implemented by TiDB(user also can implement own driver by self).
+	// import _ "github.com/pingcap/tidb/types/parser_driver" in import part.
```

Comment:
These two lines can be removed

Reply:
Step 0 doesn't take effect ? @lysu

Reply:
If we need to display these two lines on godoc.org, then the comment on the actual `import` on line 19 should be shortened as this is repeating the same information...

(The actual `import` not showing is a bug of godoc: golang/go#8223, golang/gddo#433.)

Reply:
@kennytm  yes..I tried and godoc doesn't show that so repeat it in here...- -